lib/middleware.js

```diff
@@ -89,12 +89,14 @@ module.exports = function( makeModel, apiAppModel, List ) {
       }
 
       if ( makeTags && makeTags.length ) {
-        validTags = tags.validateTags( makeTags, req.user );
+        validTags = tags.validateTags( makeTags, req.user, req.credentials.admin );
       }
 
-      // keep app tags
+      // keep app tags that aren't in ValidTags
       if ( req.make ) {
-        validTags = validTags.concat( req.make.appTags );
+        validTags = validTags.concat( req.make.appTags.filter(function( tag, pos, arr ) {
```

Comment:
this only keeps an app tag if it was included in the request body. Allows removal of app tags (allows for un-featuring) and has a helpful side effect of un-featuring a make automatically if a webmaker user updates the make for any reason.